client/src/game/types/CardState.ts

```diff
@@ -1,5 +1,6 @@
 import Color from "./Color";
 
+// "playStack" is an empty play stack, "deck" is a play stack with some cards on it
```

Comment:
its not necessarily empty

Reply:
Then why there is a need for two?

Reply:
what do you mean two?

Reply:
What do these values mean? In my tests I needed to handle both: one for case of empty stack, another for case of something on it.
What's the intended difference?